giftcard-example/src/main/resources/templates/layout.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html
+  lang="en"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+>
+  <head>
+    <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE">
+      Gift Card Demo
+    </title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+
+    <!-- Adyen JS from TEST environment (change to live for production)-->
```

Comment:
- [ ] Create an issue so we don't forget to update the E2E for gift cards and we can start migrating to the latest versions

Reply:
See #82